src/app/components/home/post-detail/post-detail.component.html

```diff
@@ -1,29 +1,58 @@
-<!-- Page content-->
-<div *ngIf="post" class="container mt-5">
+<!-- Page content start -->
+<div *ngIf="post" class="container mt-4 mb-4">
     <div class="row justify-content-center">
         <div class="col-lg-8">
-            <!-- Post content-->
+            <!-- Post content start -->
             <article>
-                <!-- Post header-->
+                <!-- Post header start -->
                 <header class="mb-4">
-                    <!-- Post title-->
+                    <!-- Post title start -->
```

Comment:
HTML comments are visible in production as well. It is recommended to keep it clean